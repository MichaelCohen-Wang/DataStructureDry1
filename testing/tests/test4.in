add_ship 64 4
get_cannons 64
add_ship 437 5
remove_ship 64
get_cannons 437
add_ship 208 19
remove_ship 437
add_pirate 174 437 2