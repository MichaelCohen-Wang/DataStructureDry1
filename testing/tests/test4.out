add_ship: SUCCESS
get_cannons: SUCCESS, 4
add_ship: SUCCESS
remove_ship: SUCCESS
get_cannons: SUCCESS, 5
add_ship: SUCCESS
remove_ship: SUCCESS
add_pirate: FAILURE
